app/middlewares/validators/user.js

```diff
@@ -37,8 +39,41 @@ const validateGetUsersDTO = (req, res, next) => {
   }
 };
 
+const validateIdentity = async (req, res, next) => {
+  try {
+    const token = utils.decodeToken(req.header(TOKEN));
+    const { error, ...user } = await UserService.getUser({ id: token.id }, ['password']);
+    if (error) {
+      throw errors.unauthorized('User does not exist in the DB.');
+    }
+    const verifyUser = utils.isObjectEqual(token, user);
+    if (!verifyUser) {
+      throw errors.unauthorized('Token data does not match DB');
+    }
+    // eslint-disable-next-line require-atomic-updates
+    req.user = user;
```

Comment:
```suggestion
    res.locals.user = user;
```
FYI http://expressjs.com/en/api.html#res.locals